Misc/NEWS.d/next/Library/2020-09-01-10-12-13.bpo-26053.hXikw_.rst

```diff
@@ -0,0 +1 @@
+Fixed args echoed by the pdb interactive run command.
```

Comment:
```rst
:mod:`pdb`
```
Also, I think this news entry needs more details.

Reply:
Thank you, I've edited it. Is this better?